definitions/procedures/content/prepare.rb

```diff
@@ -0,0 +1,11 @@
+module Procedures::Content
+  class Prepare < ForemanMaintain::Procedure
+    metadata do
+      description 'Prepare content for Pulp 3'
+    end
+
+    def run
+      execute!('foreman-rake katello:pulp3_migration')
```

Comment:
is it possible to require the pulp3 'feature' in order for this to run?

Reply:
updated! Giving it a test now on some production machines